core/emulate.c

```diff
@@ -765,6 +783,38 @@ static em_status_t decode_op_si(em_context_t *ctxt,
 }
 
 /* Soft-emulation */
+static void em_andn_soft(struct em_context_t *ctxt)
+{
+    uint64_t temp;
+    uint64_t signbit = 63;
+    temp = ~ctxt->src1.value & ctxt->src2.value;
+    if (ctxt->operand_size == 4) {
+        temp &= 0xFFFFFFFF;
+        signbit = 31;
+    }
+    ctxt->dst.value = temp;
+    ctxt->rflags = 0;
```

Comment:
Hmm, probably we should only touch the status flags (OSZAPC), since this instruction is similar to the _fastop_ instructions? Clearing other flags may have side effects, e.g. bit 1 (reserved) is supposed to be fixed to 1.